ComSemApp/models.py

```diff
@@ -363,6 +369,12 @@ def __str__(self):
     
     class Meta:
         verbose_name = "Instructor Review Request for Speaking Practice Attempt"
+    
+    def is_reviewed(self):
+        '''
+            Returns true if there is a review for this request
+        '''
```

Comment:
All the docstrings we've been making have used triple double quotes `"""`, not triple singles `'''`